deployment/apache.maintenance-template.conf

```diff
@@ -0,0 +1,24 @@
+<VirtualHost *:80>
+    ServerName evap
+
+    DocumentRoot /var/www/html
+
+    RewriteEngine on
```

Comment:
In the vagrant vm, I had to do `sudo a2enmod rewrite` once to get it working, maybe we want to do this in the provision script? (... or my vm was misconfigured :man_shrugging: )

Reply:
you are absolutely right, i forgot to add the `a2enmod`